python/ray/train/_internal/dl_predictor.py

```diff
@@ -0,0 +1,95 @@
+import abc
+from typing import Dict, TypeVar, Union, List, Tuple
+
+import numpy as np
+import pandas as pd
+
+from ray.air.util.data_batch_conversion import convert_pandas_to_batch_type, DataType
+from ray.air.util.tensor_extensions.pandas import TensorArray
+from ray.train.predictor import Predictor
+
+TensorType = TypeVar("TensorType")
+TensorDtype = TypeVar("TensorDtype")
+
+
+class DLPredictor(Predictor):
+    @abc.abstractmethod
+    def tensorize(self, numpy_array: np.ndarray, dtype: TensorDtype) -> TensorType:
```

Comment:
Should these be private?
Also nit suggestion, maybe rename to `array_to_tensor` and `tensor_to_array`?

Reply:
Updated!